src/components/HyperlinkingWrapper.js

```diff
@@ -56,11 +57,18 @@ export default class HyperlinkingWrapper extends Component {
         }
     }
 
+    onTextChange(delta, oldContents, source) {
+        const { current } = this.state;
+
+        if (current !== HYPERLINKING_STATE.EDIT && source === 'user') {
```

Comment:
would you mind to comment on why it looks like this, what does it mean and when does it happen?